api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -20,71 +27,89 @@
 
 @RunWith(SpringRunner.class)
 @DataJpaTest
-@Import({InstitutionServiceImpl.class})
+@Import({InstitutionServiceImpl.class, InstitutionMapperImpl.class})
 public class InstitutionServiceTest {
   @Autowired private InstitutionService service;
+  @Autowired private UserDao userDao;
+  @Autowired private VerifiedInstitutionalAffiliation verifiedInstitutionalAffiliation;
 
-  private final Institution testInst =
+  private final Institution TEST_INST =
```

Comment:
[style consistency] I think to use ALL_CAPS, these should be static final variables. At least that's what I'm finding from digging around our codebase for a minute.

Reply:
ok